dcompiler.dd

```diff
@@ -85,12 +85,12 @@ C:\&gt;unzip dmc.zip
 
     $(INSTALLATION_SCRIPT_HINT)
 
-    It will create
+    $(P It will create
     a $(D ~/dmd) directory with all the files in it.
 
     All the tools are command line tools, which means
     they are run from a console window.)
-    $(LI All tools can be run directly from the archive:
+    All tools can be run directly from the archive:
 $(CONSOLE
 ~/dmd/linux/bin64/dmd
```

Comment:
This was the main reason the the bug report.

Reply:
Look closer or at the generated output ;-)

Fixing the wrongly inserted parenthesis from https://github.com/dlang/dlang.org/pull/2491 makes this section Linux only as it was supposed to be.

Screenshot with this patch:

![image](https://user-images.githubusercontent.com/4370550/52314662-fbafd480-29b3-11e9-99a2-85a861e1e973.png)